daemon/src/peer/run_state.rs

```diff
@@ -594,7 +572,11 @@ mod test {
         let peer_id = PeerId::from(SecretKey::new());
         let addr = "127.0.0.0:80".parse()?;
 
-        let status = Status::Online { connected: 0 };
+        let connected_peers = one_connected_peer(PeerId::from(SecretKey::new()));
+        let status = Status::Online {
+            connected: 0,
```

Comment:
```suggestion
            connected: 1,
```

Should be equal to the number of elements in `connected_peers`.

Reply:
See below.